### ISSUE TITLE
TextEditor.get is deprecated.

### ISSUE THREAD

**Issue by alflanagan** [virtualthoughts/slickedit-select#5]
`TextEditor.prototype.displayBuffer` has always been private, but now
it is gone. Reading the `displayBuffer` property now returns a reference
to the containing `TextEditor`, which now provides *some* of the API of
the defunct `DisplayBuffer` class.
```
TextEditor.get (/usr/share/atom-beta/resources/app.asar/src/text-editor.js:101:14)
Object._handleLoad (/home/aflanagan/.atom/packages/slickedit-select/lib/slickedit-select.coffee:21:28)
<unknown> (/home/aflanagan/.atom/packages/slickedit-select/lib/slickedit-select.coffee:12:24)
Workspace.observeTextEditors (/usr/share/atom-beta/resources/app.asar/src/workspace.js:329:9)
Object.activate (/home/aflanagan/.atom/packages/slickedit-select/lib/slickedit-select.coffee:10:29)
Package.activateNow (/usr/share/atom-beta/resources/app.asar/src/package.js:191:19)
```

**Comment by virtualthoughts** [comment]
Thanks for the note!  I'm currently running with 1.12.9 and things are working... what version did this go away in?

Also if you know how to fix it feel free to send me a pull request as my day job won't leave me much time to look at this for at least the next month.

**Comment by alflanagan** [comment]
I was running atom beta 1.13, but I'll bet it now occurs in the production version. I'll send a pull request if I have time and can figure it out, but the likelihood of both of those happening is rather small, sorry.

**Comment by duanefields** [comment]
Happens with 1.14.3 production

**Comment by virtualthoughts** [comment]
Fixed in 1.6.2
